app/admin/teams.rb

```diff
@@ -1,12 +1,16 @@
 include ActiveAdminHelpers
 
 ActiveAdmin.register Team do
-  permit_params :name, user_ids: []
+  permit_params :name, :timesheet_enabled, user_ids: []
 
   form do |f|
     f.inputs 'Team Details' do
       f.input :name
       f.input :users, as: :check_boxes, collection: User.order(:email).pluck(:email, :id).map { |email, id| [email, id] }
+
+      # Add checkbox for timesheet_enabled
+      label_timesheet = f.object.timesheet_enabled ? "Disable Timesheet" : "Enable Timesheet"
```

Comment:
This is going to be confusing. 

Please set the label to be "Enable Timesheet Entries for Team". This label should never change, so it's clear what it means when it is checked versus not checked.